src/web/services/account.service.ts

```diff
@@ -118,20 +118,11 @@ export class AccountService {
   /**
    * Edits an account request by calling API.
    */
-  editAccountRequest(id: string, name: string, email: string, institute: string,
-    status: AccountRequestStatus, comments: string)
+  editAccountRequest(id: string, accountReqUpdateRequest: AccountRequestUpdateRequest)
   : Observable<AccountRequest> {
```

Comment:
This is just a nit. Would it be possible to combine this into one line? So like

```typescript
editAccountRequest(id: string, accountReqUpdateRequest: AccountRequestUpdateRequest) : Observable<AccountRequest> {
```

If I'm not wrong, the limit is 120, and this single line should only be 117, right?

Reply:
Changes Made!!
Apparently nothing went wrong, i just run the lint test 👍🏼